docs/products/flink/concepts/kafka_connectors.rst

```diff
@@ -39,3 +39,33 @@ If you want to use multiple data streams as a source, but have the results for m
 
 In addition, choose upsert connectors if you want to provide the output as a compacted topic and read only the latest value for each message key.
 
+
+Requirements for each connector type
```

Comment:
I would probably put this in a separate file under the menu item "Standard and Upsert Apache Kafka Connectors" with title "Apache Flink Connector Requirements for Apache Kafka"

Reply:
Yep, although I added the new article in parallel to avoid additional nesting in the table of contents navigation.